core/src/main/java/org/apache/accumulo/core/file/rfile/CreateEmpty.java

```diff
@@ -76,10 +79,25 @@ public void validate(String name, String value) throws ParameterException {
   }
 
   public static void main(String[] args) throws Exception {
+    new CreateEmpty().execute(args);
+  }
+
+  @Override
+  public String keyword() {
+    return "create-empty";
+  }
+
+  @Override
+  public String description() {
+    return "creates an empty rfile";
```

Comment:
```suggestion
    return "Creates an empty rfile";
```
The rest of the descriptions start with a capital letter.